.gitpod.Dockerfile

```diff
@@ -1,4 +1,26 @@
-FROM gitpod/workspace-full
+FROM gitpod/workspace-base
+
+# Installing Node
+# https://github.com/nvm-sh/nvm#installing-and-updating
+USER gitpod
+ENV NODE_VERSION=16.13.0
```

Comment:
can we use `lts/*` for our `nvm` version so we don't forget to update this? https://github.com/nvm-sh/nvm#long-term-support

Reply:
Updated the script to use LTS